common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -1111,7 +1111,7 @@ bool mon_wlan_hal_dwpal::process_dwpal_nl_event(struct nl_msg *msg)
                 return false;
             }
             // Initialize the message
-            results_buff = {};
+            memset(results_buff.get(), 0, sizeof(sCHANNEL_SCAN_RESULTS_NOTIFICATION));
```

Comment:
Are you sure that this `memset` doesn't cause a build error with UGW's GCC 8.x?

Reply:
Indeed, please make sure to build this PR in UGW with GCC8.x.
Resolve this if it passes.

Reply:
you have 40 cases of memset in the code currently so yes pretty sure.